resources/qgis_global_settings.ini

```diff
@@ -33,3 +33,8 @@ connections-xyz\OpenStreetMap\zmin=0
 # Default help location to include
 # for now this is online version of the User Guide for latest (LTR) release
 helpSearchPath=https://docs.qgis.org/$qgis_short_version/$qgis_locale/docs/user_manual/
+
+[app]
+
+# If true, the experimental "duplicate feature" actions will be shown in the QGIS UI
```

Comment:
Maybe add a mention that a restart is needed when the setting key is toggled?

Reply:
Is there any need, given that this is only shown in the ini file and a restart is implied?

Reply:
You're right, the tip would mostly be useful for IT person(s) deploying a custom .ini, which would not be relevant to switching preference on/off via the advanced settings panel.